### PR TITLE
add typescript declarations

### PR DESCRIPTION
**Pull request by willisplummer**
Hey! I've been working with your `draft-js` import and export packages, and I noticed this one didn't have its own declaration file, so I figured I'd add one. 

I followed the patterns established in `draft-js-export-html` but lmk if you have any feedback!

### CODE REVIEW

**Comment by willisplummer** [comment]
@sstur just wanted to follow up here!

File: typings/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,9 @@
+declare module 'draft-js-import-html' {
+  import draftjs = require('draft-js');
+
+  export interface Options {
+    parser(html: string): HTMLElement;
```
Comment (smvilar) [inline typings/index.d.ts:5]: @willisplummer `parser` should be optional, and `Options` should include the rest of the Options that this package forwards to the `draft-js-import-element` package:
```ts
  interface Options {
    elementStyles?: ElementStyles;
    blockTypes?: {[key: string]: string};
    customBlockFn?: (element: HTMLElement) => {type?: string, data?: BlockData}|void;
    parser?: (html: string) => HTMLElement;
  }
```

**Comment by smvilar** [comment]
Hi! I wanted to make this same PR, I'm about to send a similar PR to `draft-js-import-element`, @sstur are you maintaining this project?

**Comment by sstur** [comment]
> I'm about to send a similar PR to draft-js-import-element, @​sstur are you maintaining this project?

Yes. Would you mind making the PR to the new monorepo for this project?

https://github.com/sstur/draft-js-utils

**Comment by willisplummer** [comment]
@smvilar I'm no longer using this project - feel free to use this work as a starting point for your own PR

**Pull request closed without merging**